packages/js/product-editor/src/components/iframe-editor/header-toolbar/header-toolbar.tsx

```diff
@@ -115,19 +115,15 @@ export function HeaderToolbar( {
 					}
 					showTooltip
 				/>
-				{ isWideViewport && (
-					<>
-						{ isLargeViewport && (
-							<ToolbarItem
-								as={ ToolSelector }
-								disabled={ isTextModeEnabled }
-							/>
-						) }
-						<ToolbarItem as={ EditorHistoryUndo } />
-						<ToolbarItem as={ EditorHistoryRedo } />
-						<ToolbarItem as={ DocumentOverview } />
-					</>
+				{ isLargeViewport && (
+					<ToolbarItem
+						as={ ToolSelector }
+						disabled={ isTextModeEnabled }
+					/>
 				) }
+				<ToolbarItem as={ EditorHistoryUndo } />
+				<ToolbarItem as={ EditorHistoryRedo } />
+				<ToolbarItem as={ DocumentOverview } />
```

Comment:
This follows more inline with Gutenberg, which always shows the above toolbar items on a smaller viewport.